### PR TITLE
nawk: update to 20180823.

### PR DESCRIPTION
**Pull request by cr6git**

### CODE REVIEW

File: srcpkgs/nawk/template [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+wrksrc="awk-${_commit}"
 hostmakedepends="bison"
 short_desc="The one, true implementation of AWK"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="BSD"
-homepage="http://cm.bell-labs.com/who/bwk/"
-distfiles="http://www.cs.princeton.edu/~bwk/btl.mirror/awk.tar.gz"
-checksum=8dc092165c5a4e1449f964286483d06d0dbfba4b0bd003cb5dab30de8f6d9b83
+homepage="https://github.com/onetrueawk/awk"
+changelog="https://github.com/onetrueawk/awk/blob/master/ChangeLog"
```
Comment (maxice8) [inline srcpkgs/nawk/template]: can it be awk/raw/master ?

File: srcpkgs/nawk/template [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
-revision=5
-create_wrksrc=yes
+version=20180823
+revision=1
+_commit=0f4e1ba922ccd45f7bd47701e923d98566ceb7c5
+wrksrc="awk-${_commit}"
 hostmakedepends="bison"
 short_desc="The one, true implementation of AWK"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="BSD"
```
Comment (maxice8) [inline srcpkgs/nawk/template]: also please use SPDX identifier

Comment (cr6git) [reply srcpkgs/nawk/template]: https://github.com/onetrueawk/awk/blob/master/LICENSE
I don't know precisely... what is this license?

File: srcpkgs/nawk/template (hunk 1 above)
Comment (cr6git) [reply srcpkgs/nawk/template]: good point!

File: srcpkgs/nawk/template (hunk 2 above)
Comment (maxice8) [reply srcpkgs/nawk/template]: seems to be MIT but can't confirm

Comment (Johnnynator) [reply srcpkgs/nawk/template]: https://spdx.org/licenses/StandardML-NJ.html `This license has been deprecated since 2.0rc2.`

Comment (cr6git) [reply srcpkgs/nawk/template]: @Johnnynator looks like the right one... @maxice8 are you okay for SMLNJ ?

**Pull request merged**